### DISCUSSION TITLE
Set-up a Vanilla server with Minecollab

### DISCUSSION THREAD

**Discussion by vasava15** [thecoder-001/MineColab#76] [Q&A]
Is it possible to set-up servers without using paper or fabric? Just a purely vanilla server. I'm not the greatest with code, so any help would be appreciated.

**Comment by N-aksif-N** [comment]
In theory, yes, it is possible to make it.

Hello, If you don't might, try this: https://github.com/N-aksif-N/MineColab_Improved
It's a improvement for minecolab @vasava15
